account_payment_partner/models/account_invoice.py

```diff
@@ -22,11 +22,9 @@ class AccountInvoice(models.Model):
     def _onchange_partner_id(self):
         res = super(AccountInvoice, self)._onchange_partner_id()
         if self.partner_id:
+            self.payment_mode_id = self._get_payment_mode()
```

Comment:
Shouldn't this go out also of `if self.partner_id`?

Reply:
Looks like `self.partner_id` is set to False in the else case. So I would think not.

Reply:
indeed it can be outside, and simpler to reuse
updated